src/site/content/en/blog/receive-shared-data/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Receiving shared data
+subhead: Sharing on mobile made simple
+authors:
+  - petelepage
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  On a mobile device, sharing should be as simple as clicking the Share button,
+  choosing an app, then choosing who to share with. The Web Share Target API
+  allows installed web apps to register with the underlying operating system to receive shared content.
+tags:
+  - post
+  - capabilities
+---
+
+On a mobile device, sharing should be as simple as clicking the **Share** button,
+choosing an app, and choosing who to share with. For example, you may want to
```

Comment:
I removed all instances of first person singular since the post doesn't establish who the "I" is, and the meaning wasn't changed by using second person.